app.js

```diff
@@ -268,7 +312,7 @@ function processArgs(args, options) {
         }
       }
     })
-    .example('\t$0 --acs http://acme.okta.com/auth/saml20/exampleidp --aud https://www.okta.com/saml2/service-provider/spf5aFRRXFGIMAYXQPNV', '')
+    .example('$0 --acsUrl http://acme.okta.com/auth/saml20/exampleidp --audience https://www.okta.com/saml2/service-provider/spf5aFRRXFGIMAYXQPNV', '')
```

Comment:
I figured it was better to use full option names. Honestly, I'd rather see this named something more verbose like `--assertionConsumerServiceUrl`.

Reply:
lets stick with the short names (ok with +Url alias), i see the value in being verbose but our team uses this from CLI a lot and the short names are easier to deal with.

Reply:
I'm only using the full option name for the example, the short names still work. I think using the full names in the example makes the example easier to understand. If you disagree, I can revert the example back to the short option.